src/MetaModels/Filter/Setting/CustomSql.php

```diff
@@ -145,7 +146,7 @@ private function addParameters($parameters)
      */
     private function addParameter($parameter)
     {
-        if (empty($parameter)) {
+        if (is_null($parameter)) {
```

Comment:
To conform to code style, this should be:
```diff
- if (is_null($parameter)) {
+ if (null !== $parameter) {
```

However, this will now skip any parameter value of `null` but the corresponding `?` will still get added by the various methods.
This will for sure end in exceptions with `Too few arguments to build the query string`.
Are you really sure this works as intended on your side? If so, how?